src/scripts.js

```diff
@@ -284,6 +284,7 @@ function convertRecipeToRender(recipeToRender) {
     return name[0].toUpperCase() + name.substring(1);
   }).join(' ');
   let totalCost = convertTotalCost(recipeToRender);
+  let exitButtonClass = showHomeView();
```

Comment:
I think I see what you are going for here. I think we just need to add an event listener for this new button? and invoke show home view when its pressed?